ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -190,10 +192,16 @@ objects:
         oc create -f https://raw.githubusercontent.com/openshift/telemeter/master/deploy/telemeter-client-openshift.yaml -n openshift-telemetry
         set -e
 
-        # TODO: the test binary should really be a more structured command - most of these flags should be
-        #       autodetected from the running cluster.
-        # TODO: bump nodes up to 40 again
         function run-tests() {
+          if which openshift-tests && [[ -n "${TEST_SUITE-}" ]]; then
+            openshift-tests run "${TEST_SUITE}" --provider "${TEST_PROVIDER:-}" -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
+            exit 0
```

Comment:
Is this just while you're testing?  Won't we want the function and script to return the `openshift-tests` exit status?

Reply:
it should already be set -e